template.yaml

```diff
@@ -718,7 +719,7 @@ Resources:
         - Key: IncludedInBackup
           Value: 'true'
 
-  NvaPersonInfoTable:
+  NvaIdTypeObjectsTable:
     Type: AWS::DynamoDB::Table
     UpdateReplacePolicy: Retain
     DeletionPolicy: Delete
```

Comment:
what does nva-id-type-persistent-objects mean btw?

Reply:
it is a reference to the single-table-pattern used.
`id` and `type` as a composite key access pattern.

Reply:
Hmmm I would prefer if it have a name that reflects its content/group of content, this is in my head way to generic. if we already are changing logic id here, its a good time to change table name also.

Reply:
so evey entity can have a host of classes assosiated with it, but only one instanse of each class.

| id   | type            | createdBy | created     | modifiedBy | modified    | ...objectfields |
|------|-----------------|-----------|-------------|------------|-------------|-----------------|
| xyz  | Person          | xyz       | 10-01-2021  | xyz        | 01-01-2022  | personinfo....  |
| xyz  | Roles           | admin     | 00-01-2021  | admin      | 01-01-2023  | RolesInfo....   |
| xyz  | UserPreferences | admin     | 00-01-2021  | xyz        | 01-10-2024  | RolesInfo....   |

Reply:
probably do a specific table name in a later pr